### DISCUSSION TITLE
如何加入“攻读学位期间取得的研究成果”，使其出现在目录中，但不对其做章节编号？

### DISCUSSION THREAD

**Discussion by CloudyDory** [stone-zeng/fduthesis#297] [Q&A] [answered]
尝试了在main.tex中加入：`\chapter{攻读学位期间取得的研究成果}`，但这样会对其作为单独一章并编号，请问如何将章节号去掉？

**Comment by note286** [comment]
`\chapter*{}`

**Comment by atxy-blip** [comment]
```tex
\begingroup
\ctexset{chapter/numbering=false}
\chapter{攻读学位期间取得的研究成果}
\endgroup
```
~~其实 `fduthesis` 提供了实现这个功能的内部命令 `\__fdu_chapter:n`，但不知为何曾老师没有做用户接口（摊手~~

私密马赛，请看曾老师的回复。

**Comment by stone-zeng** [comment, marked as answer]
请写在 `\backmatter` 之后，这样 `\chapter` 就不会编号了，并不需要 hack……

**Reply by phyzhenli** [reply]
请问这里手写时怎么修改英文字号？
```
\chapter{攻读学位期间研究成果}

\noindent
\fontsize{18pt}{18pt}{第一作者(含共同一作)：}
\vspace{5pt}

\fontsize{30pt}{18pt}{
  [1] test
  
  [2] test2
}
```

这里`第一作者(含共同一作)：`可以正常修改字体大小，然而\fontsize对下面的`[1] test [2] test2`不起作用，谢谢。


![image](https://github.com/stone-zeng/fduthesis/assets/48823046/3d3754a1-1bac-4259-b459-a8b2048b52c5)

**Reply by atxy-blip** [reply]
@phyzhenli 因为 `\fontsize` 做的修改最终还要靠 `\selectfont` 才能生效，而且要和被修改的内容放在一个 group 内。

此外这个实际上是字号问题，跟本贴主题无关，下次建议单独开贴提问。

```tex
\documentclass{fduthesis}

\begin{document}
\backmatter
\chapter{攻读学位期间研究成果}

\noindent
{
\fontsize{18pt}{18pt}\selectfont
第一作者(含共同一作)：
}

作者

\vspace{5pt}
{
\fontsize{30pt}{18pt}\selectfont
  [1] test
  
  [2] test2
}
\end{document}
```
